src/jit-diff/diff.cs

```diff
@@ -515,8 +515,8 @@ protected override List<string> ConstructArgs(List<string> commandArgs, string c
 
             void InstallBaseJit()
             {
-                string realJitPath = Path.Combine(m_config.CoreRoot, m_defaultJitName);
-                string tempJitPath = Path.Combine(m_config.CoreRoot, "backup-" + m_defaultJitName);
+                string realJitPath = Path.Combine(m_config.CoreRoot, m_testJitName);
```

Comment:
What if the install doesn't have a jit by this name? Seems like this could happen for some, maybe new, altjit.

Seems like "tempJitPath" should be "backupJitPath".

Reply:
Yeah, probably warrants some more complex logic.